.github/workflows/ci.yml

```diff
@@ -36,7 +36,10 @@ jobs:
       - uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
         with:
           go-version: "1.22"
-      - run: make integration-tests
+      - name: Install gingko
```

Comment:
I am experimenting with something atm. Kubebuilder generates a Makefile that installs the needed tools in a `bin` directory in the project root.  We could add ginkgo to the automatically installed tools, cache the bin directory, and invalidate the cache if the Makefile changes. WDYT?

Reply:
That sounds fine to me, provided the tools versions are pinned in the Makefile. We have that layout in other repos.
I would prefer a GH action because they can or tend to verify the tools prior to install.